nomenclature/definitions/variable/technology/electrity-operation.yaml

```diff
@@ -15,66 +15,90 @@
+   unit: MWh
+   
+Active Power|Electricity|Biomass|Traditionnal:
    description: Active power of  Traditionnal Biomass power plants
                 biomass from local and pre-existing
                 resources mostly waste :domestic and agricultural waste, forest residues.
    unit: MWh
 
-Active power|Electricity|Biomass|New and imported:
+Active Power|Electricity|Biomass|New and imported:
    description: Active power of New+Imported biomass power plants
                 biomass from dedicated energy crops and biomass imports.
    unit: MWh
 
-Active power|Electricity|Nuclear:
+Active Power|Electricity|Nuclear:
    description: Active power of nuclear power plants
    unit: MWh
 
-Active power|Electricity|Gas|OCGT:
+Active Power|Electricity|Gas:
+   description: Active power of Gas power plants
+   unit: MWh
+   
+Active Power|Electricity|Gas|OCGT:
    description: Active power of Open Cycle Gas Turbine power plants
    unit: MWh
+   
+Active Power|Electricity|Gas|CCGT:
+   description: Active power of Combined Cycle Gas Turbine power plants
+   unit: MWh
 
-Active power|Electricity|Gas|Coal:
+Active Power|Electricity|Coal:
    description: Active power of coal power plants
    unit: MWh
 
-Active power|Electricity|Lignite:
+Active Power|Electricity|Lignite:
    description: Active power of Lignite power plants
    unit: MWh
+
+Active Power|Electricity|Oil:
+   description: Active power of oil power plants
+   unit: MWh   
+
+Active Power|Electricity|Geothermal:
+   description: Active power of Geothermal power plants
+   unit: MWh
    
-Active power|Electricity|Hydro|Reservoir:
+Active Power|Electricity|Hydro|Reservoir:
    description: Active power of "simple Hydro reservoir " units
    unit: MWh
    
-Active power|Electricity|Hydro|Pumped Storage:
+Active Power|Electricity|Hydro|Pumped Storage:
    description: Active power of "Pumped Storage " units
    unit: MWh
    
-Active power|Electricity|Hydro|Run of River:
+Active Power|Electricity|Hydro|Run of River:
    description: Active power of "Run of River" units
    unit: MWh
+
+Active Power|Electricity|Hydro|Ocean:
+   description: Active power of Ocean power plants
+   unit: MWh
    
-Active power|Electricity|Solar:
+Active Power|Electricity|Solar:
    description: Active power of Photovoltaic power plants
    unit: MWh
    
-Active power|Electricity|Wind|OnShore:
+Active Power|Electricity|Wind|OnShore:
    description: Active power of  Wind Onshore power plants
    unit: MWh
    
-Active power|Electricity|Wind|OffShore:
+Active Power|Electricity|Wind|OffShore:
    description: Active power of  Wind OffShore power plants
    unit: MWh
    
-Active power|Electricity|Load Curtailement:
+Active Power|Electricity|Load Curtailement:
    description: Energy curtailed - from the system point of view is equivalent to generation of Active Power
    unit: MWh
    
-Active power|Electricity|Load Shifting:
+Active Power|Electricity|Load Shifting:
```

Comment:
Are both curtailment and load shifting seen as  "eqivalent to generation of active power"? Does this mean that load shifting can take negative values when demand is increased?

As an aside: I think this fits well with what I now proposed for demand flexibility potentials, namely:
Load Shifting Potential|Electricity|Residential|AC

Reply:
Yes it is, can take negative values. For load shifting if the model decides to move load from instant t1 to instant t2 then  what happens is that -the electricity demand variable does not change  -load shifting(t1) is positive (which is equivalent to decreasing the elec load) - load shifting in t2 is negative, which is equivalent to increasing the elec load
Nadia is working on defining additionnal input variable for feeding the model with also the related constraints (eg time horizon constraints for the shifting....)

Reply:
Great, thanks!
looks good to me for a merge with Master.